Sources/Views/MessagesCollectionView.swift

```diff
@@ -65,7 +65,20 @@ open class MessagesCollectionView: UICollectionView {
 
     public func scrollToBottom(animated: Bool = false) {
         guard let indexPath = indexPathForLastItem else { return }
-        scrollToItem(at: indexPath, at: .bottom, animated: animated)
+        
+        let collectionViewContentHeight = collectionViewLayout.collectionViewContentSize.height
+        let isContentTooSmall = (collectionViewContentHeight < bounds.height * 2)
```

Comment:
Why `bounds.height * 2` here?

Reply:
The reason is `scrollToItem` would sometimes invalid when `collectionViewContentSize`'s `height` slightly larger than `collectionView`'s `height`.

Reply:
Yes, I understand that, but why did we decide to use `* 2`? Is this just a random number?

Reply:
Nope, it's based on my tests, I found `scrollToItem` sometimes invalid when last cell's maxY greater than bounds's height but less than 2 * bounds's height.